aten/src/ATen/cpu/vec/vec256/vec256_bfloat16.h

```diff
@@ -1049,7 +1048,35 @@ inline Vectorized<type> convert_float_##name(const Vectorized<float>& a, const V
   return Vectorized<type>::loadu(arr2); \
 }
 CONVERT_NON_VECTORIZED_INIT(BFloat16, bfloat16);
+#if defined(__aarch64__) && !defined(C10_MOBILE) && !defined(__CUDACC__)
+inline std::tuple<Vectorized<float>, Vectorized<float>> convert_half_float(const Vectorized<Half>& a) {
+  constexpr int64_t K = Vectorized<Half>::size();
+  __at_align__ float16_t arr2[K];
+  a.store(arr2);
```

Comment:
Why do we need to do this? Can you use this https://fburl.com/code/t13r938d?

Reply:
Good point (I though that store is not guaranteed to be aligned, but it is, so let me make a change here)